src/components/validationStatus.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div>
+    <div class="validation-container" v-if="toggleValidationPanel">
+      <span class="validation-container__defaultMessage" v-if="!numberOfValidationErrors">no problems to report</span>
+      <div class="validation-container__list"  v-for="error in errorList" :key="error.id">
+        <span class="validation-container__list--id">{{ error.id }}</span>
+        <span class="validation-container__list--message">
+          <span class="validation-container__list--key">{{ error.errorKey }}</span>
+          <span>{{ error.message }}</span>
+        </span>
+        <span class="validation-container__list--errorCategory"
+              :style="[ error.category === 'error'
```

Comment:
Instead of using `style`, I think it might be cleaner to put this into a BEM modifier class, and pass in a method for getting the class name instead of using a ternary in the template.